docs/4-return-of-the-monitoring/1-enable-monitoring.md

```diff
@@ -117,29 +130,26 @@
 
 > Let's extend the Pet Battle Dashboard with a new `panel` to capture some metrics in a visual way for us. Configuring dashboards is easy through the Grafana UI. Then Dashboards are easily shared as they can be exported as a `JSON` document.
 
-1. OpenShift users have a read-only view on Grafana by default - get the `admin` user details from your cluster:
```

Comment:
I assume we switched to OpenShift OAuth, that's why this step is no longer needed, is that correct? 🤔